rep-2001.rst

```diff
@@ -1,6 +1,6 @@
 REP: 2001
 Title: ROS 2 Variants
-Author: Tully Foote <tfoote@openrobotics.org>, Mikael Arguedas <mikael@openrobotics.org>, Ruffin White <roxfoxpox@gmail.com>
+Author: Tully Foote <tfoote@openrobotics.org>, Mikael Arguedas <mikael@openrobotics.org>, Ruffin White <roxfoxpox@gmail.com>, David V. Lu!! <davidvlu@gmail.com>
```

Comment:
Y'all should add your names here.

Reply:
Done in [70575c9](https://github.com/ros-infrastructure/rep/pull/355/commits/70575c978b0162ad5ca1b50c4c27481fbd75df96)